package.json

```diff
@@ -4,21 +4,22 @@
   "private": true,
   "homepage": "https://poanetwork.github.io/",
   "dependencies": {
+    "ajv": "^6.5.2",
```

Comment:
@varasev why do we need it? where do we use it?

Reply:
@vbaranov `ajv` (json schema validator) is used by several packages such as `webpack`, `eslint`, etc. When I was updating packages, I saw npm's warning that `ajv` version `^6.x.x` is recommended, so I decided to install `ajv` as well.

Reply:
ok, thanks for the clarification